### ISSUE TITLE
The "Drugs" tag is poorly defined

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/contentdb#540]
The "Drugs" tag definition reads as follows:

> Contains illegal drugs, such as marijuana

This is wrong. Marijuana isn't illegal everywhere. So this tag is biased towards countries that make it illegal.

I think the definition of the "Drugs" tag needs to be better, more well-defined and most-importantly, neutral.

We have a similar tag called "Alcohol / Tobacco". This is well-defined and doesn't need any change.

## Solution

I suggest the following changes:

* Change the description of "Drugs" to "Contains drugs other than alcohol or tobacco"
* Change the tag name "Drugs" to "Drugs (not alcohol/tobacco)"

Yes, the new tag name is longer but the term "Drugs" alone is too vague and may be misleading. Alcohol and tobacco are drugs, too, so this is irreducible complexity. This tag is used as a content warning, so I think clarity is more important than brevity.

## Alternative solution

Another option would be to redesign all drugs-related tags from the ground-up but this means all packages have to be carefully re-reviewed and re-tagged and I don't know if this is worth the effort.

**Comment by rubenwardy** [comment]
ContentDB's content warnings are largely based on PEGI warnings. Here's the one that Alcohol/Tabacco and Drugs came from.

![image](https://github.com/minetest/contentdb/assets/2122943/f5908d96-7f5f-47b4-87ba-a78b6296ac1a)

I've changed the description to "Contains recreational drugs other than alcohol or tobacco"

**Comment by Wuzzy2** [comment]
Well, in that case, PEGI sucks. :-)
As I said, the term "illegal drugs" is a weasel word because there are about 190 countries worldwide with different laws and regulations.

I like your new definition, it is good. What do you think about my new proposed tag name itself?

**Comment by Dragonop** [comment]
I disagree on renaming the tag itself, since by being overly specific the CDB staff lose versatility when applying content warnings

My feel is that the new description is sufficiently graphic in making sure people understand it referes to the "harder" drugs, and not stuff like tobacco, alcohol, or even coffee or medicine

That said, and most importantly, renaming to "Drugs (not alcohol/tobacco)" is both redundant with the "Alcohol / Tobacco" tag and with the new description.
